### PR TITLE
Include backport branches in publishing

### PR DESCRIPTION
**Pull request by drostron**

### CODE REVIEW

File: scripts/publishJarIfMaster [hunk 1]
```diff
@@ -15,7 +15,9 @@ if [[ ! -v GITHUB_TOKEN ]] ; then
 fi
 
 # only publish on quasar-analytics/quasar#master
```
Comment (alissapajer) [inline scripts/publishJarIfMaster]: We should probably update the comment now that we publish backport as well.

File: scripts/publishJarIfMaster [hunk 2]
```diff
@@ -15,7 +15,9 @@ if [[ ! -v GITHUB_TOKEN ]] ; then
...
-if [[ "$TRAVIS" == "true" && "$TRAVIS_BRANCH" == "master" && "$TRAVIS_REPO_SLUG" == "quasar-analytics/quasar" ]] ; then
+if [[ "$TRAVIS" == "true"                                              &&
+      ("$TRAVIS_BRANCH" == "master" || "$TRAVIS_BRANCH" == backport/*) &&
```
Comment (alissapajer) [inline scripts/publishJarIfMaster]: Is there a reason `"master"` is in quotes while `backport/*` is not?

File: scripts/publishJarIfMaster (hunk 1 above)
Comment (drostron) [inline scripts/publishJarIfMaster]: Sure. The filename is out of date now too. Will update accordingly.

File: scripts/publishJarIfMaster (hunk 2 above)
Comment (drostron) [inline scripts/publishJarIfMaster]: The prefix matching only works outside of quotes.

Comment (alissapajer) [inline scripts/publishJarIfMaster]: Ok cool, I figured as such, just wanted to double check that it was intentional.

File: scripts/publishJarIfMaster (hunk 1 above)
Comment (alissapajer) [inline scripts/publishJarIfMaster]: Sounds good. After that commit lands I will merge!

**Comment by drostron** [comment]
Realized we'll need to update sbt-slamdata's `publishAndTag` similarly. Will include that here once available.

**Comment by alissapajer** [comment]
@drostron when you've pushed everything you need to, just let me know, and I'll take another look!

**Comment by drostron** [comment]
@alissapajer Ready for another look

File: project/build.sbt [hunk 3]
```diff
@@ -2,7 +2,7 @@ libraryDependencies += "org.kohsuke" % "github-api" % "1.59"
 
 disablePlugins(TravisCiPlugin)
 
-scalacOptions ++= commonScalacOptions_2_10
+scalacOptions ++= scalacOptions_2_10
```
Comment (alissapajer) [inline project/build.sbt]: Why did this change now?

Comment (drostron) [inline project/build.sbt]: It comes with the sbt-slamdata 0.1.0 update

**Pull request merged**